molgenis-data-postgresql/src/test/java/org/molgenis/data/postgresql/PostgreSqlExceptionTranslatorTest.java

```diff
@@ -111,7 +111,7 @@ public void translateDependentObjectsStillExistMultipleDependentTables()
 		MolgenisValidationException e = postgreSqlExceptionTranslator
 				.translateDependentObjectsStillExist(new PSQLException(serverErrorMessage));
 		assertEquals(e.getMessage(),
-				"Cannot delete entity 'myEntity' because entities 'myRefEntity, myOtherRefEntity' depend on it.");
+				"Cannot delete entity 'myRefEntity' because entity 'myEntity' depends on it..Cannot delete entity 'myOtherRefEntity' because entity 'myEntity' depends on it.");
```

Comment:
This is how the message of a MolgenisValidationException with two constraint violations with messages `xxxxxxx.` and `yyyyyyy.` is constructed.